server.go

```diff
@@ -226,10 +231,12 @@ func validIP4(ipAddress string) bool {
 }
 
 // Serve starts and listens RPC requests.
-//It is blocked until receiving connectings from clients.
-func (s *Server) Serve(network, address string) (err error) {
+// It is blocked until receiving connectings from clients.
+// The blockCrypt argument can be nil if the caller
+// wishes to skip encryption, or is not using the kcp transport.
+func (s *Server) Serve(network, address string, blockCrypt kcp.BlockCrypt) (err error) {
```

Comment:
kcp is a plugin and few uses will use it in production. It should not be a parameter here.